sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -14,6 +16,10 @@
   @NotNull
   ISpan startChild(@NotNull String operation);
 
+  @ApiStatus.Internal
+  @NotNull
+  ISpan startChild(@NotNull String operation, @NotNull Date timestamp);
```

Comment:
`l`: Maybe to algin with the other methods?
```suggestion
  ISpan startChild(@NotNull String operation, @NotNull Date startTimestamp);
```

Reply:
a child (Span) has a timestamp, a transaction has a startTimestamp, not the same thing